docs/content/preview/releases/ybdb-releases/end-of-life/v2.19.md

```diff
@@ -6,16 +6,32 @@ description: Enhancements, changes, and resolved issues in the v2.19 release ser
 aliases:
   - /preview/releases/release-notes/latest-release/
```

Comment:
should these aliases not be deleted. check other eol files and remove any aliases that were copied over; they should only be on the real preview release